docs/releases/1.11.txt

```diff
@@ -258,6 +258,9 @@ Tests
 * Added :meth:`.DiscoverRunner.get_test_runner_kwargs` to allow customizing the
   keyword arguments passed to the test runner.
 
+* Added ``debug`` keyword argument to
```

Comment:
I'm on the fence about the release note. This seems more useful for Django internally than for any third-party tools, but I suppose a third-party test runner might make use of it. However, like the note right about, this note should have a short description about the usefulness of the feature.